### PR TITLE
removes moth language

### LINKED ISSUE #34736: AI's cannot understand Mothspeak.
Issue reported from  Round ID: 82296 (/tg/Station Bagil [ENGLISH] [US-EAST] [100% FREE LAG])

Unlike Draconian (lizardperson language), AI's are unable to understand the snowflake lanugage. This becomes a problem when you are unable to learn of the moth's attempt at revolution.

### PR DESCRIPTION
**Pull request by praisenarsie**
there was no need to give them a separate language, they already have a flight in low grav which is a pretty good feature. plus the only good race language is  draconic which are given to lizards and ash lizards because it makes sense lore-wise. (infact LIZARDS HAVE LORE AND MOTHS DON'T, MOTHS DON'T EVEN HAVE A WIKIPAGE)

infact i'm surprised no one has added anything related to moths on the wiki, it makes me think people only wanted this to spread their furry ideals.

also no this isn't obvious bait, i like to think moths are related to flypeople and that to become a moth, you were mutated into one and since flypeople don't get their own language, i'm making this pr

also i'm keeping the language icon there just in case.

@KorPhaeron 

Fixes #34736 
Closes #34798

### CODE REVIEW

**Comment by ayyourboibilly** [comment]
fly people are a reference to a old movie but  moth people are obviously bagil furry memes

**Comment by coiax** [comment]
If the moths were tribal people from Planetstation, then why wouldn't they have their own language?

People complained BITTERLY when lizards got their own language, but it didn't actually change much, apart from letting lizards smacktalk the humans.

**Comment by praisenarsie** [comment]
> If the moths were tribal people from Planetstation, then why wouldn't they have their own language?

except planetstation doesn't exist yet...

**Comment by MoreRobustThanYou** [comment]
well i'm the one who PR'd moths here and I say they should get a language to shit talk non-moths

**Comment by praisenarsie** [comment]
sorry you PORTED moths and made it so they appeal to furries

**Comment by MoreRobustThanYou** [comment]
i did not, you sick fuck. furries should be wiped from ss13

**Comment by DaxDupont** [comment]
>moths don't have a wiki page yet

Have you considered the fact that nobody made one yet? Lizards didn't have a wiki page from the first month they were merged. That all came later.

**Review by Cyberboss — DISMISSED** [review]

**Comment by 81Denton** [comment]
>infact i'm surprised no one has added anything related to moths on the wiki,

Nobody ever updates the fucking wiki, have you seen recent changes? It's probably five people in total who try to keep it up to date.

**Comment by Armhulen** [comment]
ok fine i'll make a wiki page

god

**Comment by tortellinitony** [comment]
species should be cosmetic

moths have been ruined

**Comment by FuryMcFlurry** [comment]
should I write an essay of a fan fiction where mothfolk came from?

It all started as a meme 3 years ago when people were moth posting and wanting moth race. I made them and added them onto hippie. /ss13g/ was salty afterwards

**Comment by praisenarsie** [comment]
@FuryMcFlurry accurate

**Comment by Buggy123** [comment]
I maintain my stance that moths are cute and fine as-is, aside from the horrendous lack of screaming.

**Comment by Xhuis** [comment]
🤦‍♂️

**Comment by Buggy123** [comment]
What? I can't tell what that emoticon is.

**Comment by Xhuis** [comment]
It's a dude facepalming. I was responding to the main PR, not your comment

**Comment by Grazyn** [comment]
Why does everything needs a complex backstory? They're moth-like aliens. They come from moth planet

**Comment by Buggy123** [comment]
Look moths are cute okay just let them be.

**Comment by kevinz000** [comment]
@MoreRobustThanYou you initially adding something doesn't mean you should get absolute control of it.

**Comment by Kromgar** [comment]
Lizards have a fetish gimmick remove their language

**Comment by DaedalusGame** [comment]
i will create this pr as soon as i wake up

**Comment by davethwave** [comment]
It's easy to make a universal translator via circuits so basically when I am on and I can get a circuit printer I usually just stick one in a closet and a radio so whenever someone speaks non spacemen chat it gets translated into space man and sent back into common chat.

**Comment by PsyCommando** [comment]
You know guys, maybe just looking at this without putting some unrelated agenda/cause behind it would help a bit making a choice.. I mean, I really don't know, but at the end of the day when you make change to a game, you usually want to make it objectively better for all players. Not better only for you, or just to spite people. Just saying..

With that said. Why not just let them keep their language if its already there? Its less efforts.. Plus. nearly everything that's non-human has its own language already anyways. And the argument that they don't have a lore and thus shouldn't need a language is kinda self-contradicting. I mean "They don't have much lore, so lets remove part of their lore!"?

Also, I wouldn't say a language on TG is a gimmick. Its more like a gameplay mechanic. I guess I tend to see it as a sort of fancy chat filter.

**Comment by AlexTheSysop42** [comment]
keep the moth language if even just so people can make translators to annoy mothpeople

**Comment by Spaceinaba** [comment]
while we're at it with nerfing moths can somebody go slow down their flight, magboots that let you run full speed just for being a moth is not the best in terms of balance so it should be pretty slow considering you know, they're giant moth dudes

**Comment by DaedalusGame** [comment]
the "magboots" only work for nograv and have none of the properties that normal magboots have tho

**Comment by Spaceinaba** [comment]
im just saying it seems pretty absurd to just let them usain bolt around in zero gravity while every other race is slow

**Comment by ExcessiveUseOfCobblestone** [comment]
The own language should be lizard's thing.

**Comment by KorPhaeron** [comment]
Moth flight is actually a worse offender than the language yeah

File: code/modules/mob/living/carbon/human/species_types/mothmen.dm [hunk 1]
```diff
@@ -21,11 +21,9 @@
 		if(!H.dna.features["moth_wings"])
 			H.dna.features["moth_wings"] = "[(H.client && H.client.prefs && LAZYLEN(H.client.prefs.features) && H.client.prefs.features["moth_wings"]) ? H.client.prefs.features["moth_wings"] : "Plain"]"
 			handle_mutant_bodyparts(H)
-	C.grant_language(/datum/language/moth)
 
 /datum/species/moth/on_species_loss(mob/living/carbon/C)
 	. = ..()
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/mothmen.dm]: Remove the proc now that it's empty

**Review by Cyberboss — DISMISSED** [review]

**Comment by Qbopper** [comment]
>it makes me think people only wanted this to spread their furry ideals.

op is retarded, rip

**Comment by DaedalusGame** [comment]
ok its treason then

**Comment by AristoVagrant** [comment]
Why are you like this?

**Comment by KomradeSpectre** [comment]
I'd like to mention that Izimann has been actively opposed to moths ever since their introduction. This may have been the wrong person to evaluate and merge this PR.

**Comment by MoreRobustThanYou** [comment]
revert this

**Comment by MoreRobustThanYou** [comment]
@Cyberboss @ninjanomnom @ShizCalev @KorPhaeron Please revert this, as @lzimann is EXTREMELY biased against moths

**Comment by Qbopper** [comment]
even if it stays merged at least let someone who isn't lzimann confirm that it should stay merged

lzimann has consistently shown insane bias and wordlessly closed prs he doesn't like and it's just moronic at this point to not acknowledge it

**Comment by Xhuis** [comment]
this PR was made and merged in bad faith, and I hope that the other maintainers will give some sort of input on it
@ChangelingRain you were the one who created the PR where moths were originally merged and the other person who worked on them (MRTY) has given their input, what are your thoughts?

**Comment by sawrge** [comment]
The moth language is terrible thank you for this PR

**Comment by Qbopper** [comment]
just want to make it clear that i'm not in favor of moths having their own language but \>mfw lzimann speedmerges a pr like this

**Comment by lzimann** [comment]
i wouldn't consider 24h+ a speedmerge, but sure, histories gotta fit in people's personal narratives don't they?

**Comment by Qbopper** [comment]
i'm blind and can't read the time it was posted

that being said don't act like you don't have clear and obvious biases please

**Comment by McBawbaggings** [comment]
what happened to people picking races for fun gimmicks rather than pros/cons
why do races have to be balanced in such a way instead of letting players just play the race because of the roleplay opportunities it brings

**Comment by Qbopper** [comment]
if there was no negative downside to a nonhuman race then people who powergame would choose it - eg. being able to fly in (pressurized) zero g with no negative would be a straight upgrade

mind you the loss of law 2 is a big deal, but whatever

roleplay opportunities are few and far between nowadays and nonviolent interactions between people on the station are (compared to the past) very rare so people are going to approach things from that angle

**Comment by McBawbaggings** [comment]
hence the use of pros/cons
non-humans are functionally the same on hippie outside of AI laws but are still protected from random grief from sillycones.
we have people who play skeletons because they're funny despite them offering no ingame advantages

**Comment by Qbopper** [comment]
we don't let silicons just kill every nonhuman FNR either

there's a large population on the forums/github who have an "anti furry" mentality where nonhuman races like lizards/catpeople/moths are from people "pushing an agenda", I guess? the majority of the playerbase doesn't really care

I guess the attempt to keep them "balanced" is supposed to appease people but I don't know or really care I just play humans tbh

**Pull request merged**